specification/keyvault/resource-manager/Microsoft.KeyVault/stable/2022-11-01/examples/purgeDeletedVault.json

```diff
@@ -0,0 +1,16 @@
+{
+  "parameters": {
+    "vaultName": "sample-vault",
+    "location": "westus",
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "api-version": "2022-11-01"
+  },
+  "responses": {
+    "200": {
+      "headers": {}
+    },
+    "202": {
+      "headers": {}
+    }
```

Comment:
Long running operation should return location or azure-AsyncOperation in header but not provided

Reply:
Fixed